deploy/packer/base/scripts/packages.sh

```diff
@@ -18,7 +18,7 @@ sudo sh -c "echo deb https://get.docker.io/ubuntu docker main\
 sudo apt-get update -y
 
 ## Install all the packages
-sudo apt-get install -y lxc-docker btrfs-tools lvm2 curl build-essential \
+sudo apt-get install -y lxc-docker-1.6.1 btrfs-tools lvm2 curl build-essential \
```

Comment:
pin version for now, subsequent PR coming soon will be pinning this, the vagrant image, and CI image to >=1.7.0